src/lib/components/attributes/attribute-overview/AttributeOverview.svelte

```diff
@@ -41,16 +41,32 @@
 		pointsRemaining--;
 	};
 
-	// eslint-disable-next-line @typescript-eslint/no-empty-function
-	const randomize = () => {};
+	const randomize = () => {
+		const allocation = { ...initialAllocation };
+		const attributeNames = Object.values(ATTRIBUTES);
+
+		for (let count = 0; count < POINTS_TO_DISTRIBUTE; count++) {
+			const randomAttribute = attributeNames[(Math.random() * attributeNames.length) | 0];
+			allocation[randomAttribute]++;
+		}
+
+		pointsRemaining = 0;
+		value = allocation;
+		displayedPointsRemaining.set(pointsRemaining);
+		displayedAttributes.set(allocation);
+	};
```

Comment:
This contains some of the solution code, but these two values are not set further up here.

```suggestion
	};
```